code/__DEFINES/preferences.dm

```diff
@@ -59,10 +59,11 @@
 #define PREFTOGGLE_2_SEE_ITEM_OUTLINES 	4096
 #define PREFTOGGLE_2_HIDE_ITEM_TOOLTIPS  8192
 #define PREFTOGGLE_2_THOUGHT_BUBBLE	16384
+#define PREFTOGGLE_2_DANCE_DISCO	32768
 
-#define TOGGLES_2_TOTAL 			32767 // If you add or remove a preference toggle above, make sure you update this define with the total value of the toggles combined.
+#define TOGGLES_2_TOTAL 			65535 // If you add or remove a preference toggle above, make sure you update this define with the total value of the toggles combined.
 
-#define TOGGLES_2_DEFAULT (PREFTOGGLE_2_FANCYUI|PREFTOGGLE_2_ITEMATTACK|PREFTOGGLE_2_WINDOWFLASHING|PREFTOGGLE_2_RUNECHAT|PREFTOGGLE_2_DEATHMESSAGE|PREFTOGGLE_2_EMOTE_BUBBLE|PREFTOGGLE_2_SEE_ITEM_OUTLINES|PREFTOGGLE_2_THOUGHT_BUBBLE)
+#define TOGGLES_2_DEFAULT (PREFTOGGLE_2_FANCYUI|PREFTOGGLE_2_ITEMATTACK|PREFTOGGLE_2_WINDOWFLASHING|PREFTOGGLE_2_RUNECHAT|PREFTOGGLE_2_DEATHMESSAGE|PREFTOGGLE_2_EMOTE_BUBBLE|PREFTOGGLE_2_SEE_ITEM_OUTLINES|PREFTOGGLE_2_THOUGHT_BUBBLE|PREFTOGGLE_2_DANCE_DISCO)
```

Comment:
I'm going to need to run an update when this is merged to auto enable this for existing people

@ParadiseSS13/commit-access DNM without asking me if I am around first so I can do the above